src/test/java/org/gridsuite/study/server/utils/elasticsearch/DisableElasticsearch.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.study.server.utils.elasticsearch;
+
+import org.gridsuite.study.server.elasticsearch.EquipmentInfosService;
+import org.gridsuite.study.server.elasticsearch.StudyInfosService;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.mock.mockito.MockBeans;
+import org.springframework.test.context.TestPropertySource;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@TestPropertySource(properties = {"spring.data.elasticsearch.repositories.enabled=false"})
+@MockBeans({@MockBean(EmbeddedElasticsearch.class), @MockBean(StudyInfosService.class), @MockBean(EquipmentInfosService.class)})
```

Comment:
Could mocking StudyInfosService and EquipmentInfosService here be a source of bug ?

Reply:
if we turn off ES, these 2 beans cannot be used